app/utils/estimate_prioritization/estimate_prioritization_test.py

```diff
@@ -0,0 +1,396 @@
+            'serum_pos_prevalence': 0.1,
+            'denominator_value': 100,
+            'subgroup_var': 'Primary Estimate',
+            'academic_primary_estimate': None,
+            'dashboard_primary_estimate': None,
+            'pop_adj': True,
+            'test_adj': True,
+            'isotypes_reported': ['IgG', 'IgM'],
+            'isotype_comb': 'OR',
+            'test_type': 'ELISA',
+            'specimen_type': 'Dried Blood',
+            'age': 'All',
+            'sex': 'ALL',
+            'TEST_INFO_AVAIL': False
+        },
+        {
+            'IDENTIFIER': 1,
+            'study_name': 'specimen type',
+            'serum_pos_prevalence': 0.4,
+            'denominator_value': 200,
+            'subgroup_var': 'Primary Estimate',
+            'academic_primary_estimate': None,
+            'dashboard_primary_estimate': None,
+            'pop_adj': True,
+            'test_adj': True,
+            'isotypes_reported': ['IgG', 'IgM'],
+            'isotype_comb': 'OR',
+            'test_type': 'ELISA',
+            'specimen_type': 'Not Dried Blood',
+            'age': 'All',
+            'sex': 'ALL',
+            'TEST_INFO_AVAIL': False
+        },
+        {
+            'IDENTIFIER': 0,
+            'study_name': 'dynamic adjustment',
+            'serum_pos_prevalence': 0.1,
+            'denominator_value': 100,
+            'subgroup_var': 'Primary Estimate',
+            'academic_primary_estimate': None,
+            'dashboard_primary_estimate': None,
+            'pop_adj': True,
+            'test_adj': True,
+            'isotypes_reported': ['IgG', 'IgM'],
+            'isotype_comb': 'OR',
+            'test_type': 'ELISA',
+            'specimen_type': 'Dried Blood',
+            'age': 'All',
+            'sex': 'ALL',
+            'TEST_INFO_AVAIL': False
+        },
+        {
+            'IDENTIFIER': 1,
+            'study_name': 'dynamic adjustment',
+            'serum_pos_prevalence': 0.4,
+            'denominator_value': 200,
+            'subgroup_var': 'Primary Estimate',
+            'academic_primary_estimate': None,
+            'dashboard_primary_estimate': None,
+            'pop_adj': True,
+            'test_adj': False,
+            'isotypes_reported': ['IgG', 'IgM'],
+            'isotype_comb': 'OR',
+            'test_type': 'ELISA',
+            'specimen_type': 'Not Dried Blood',
+            'age': 'All',
+            'sex': 'ALL',
+            'TEST_INFO_AVAIL': False
+        },
+    ]
+
+    # Adding these hardcoded values for now to make tests pass
+    for estimate in sample_estimates:
+        estimate['population_group'] = "General Population"
+        estimate['state'] = ["Texas"]
+        estimate['city'] = ["Dallas"]
+        estimate['estimate_grade'] = 'National'
+
+    sample_df = pd.DataFrame(sample_estimates)
+    # Adding these hardcoded values for now to make tests pass
+    sample_df['sampling_start_date'] = datetime.datetime.now()
```

Comment:
might be simpler to add in the other hardcoded values at this point too - you could nix the for loop if you do it on a dataframe level